src/models/db.js

```diff
@@ -84,6 +95,13 @@ export default config => {
     }
   });
 
+  hooks.forEach(hook => {
+    const clientHook = config.hooks[hook];
+    if (clientHook) {
+      sequelize.addHook(hook, clientHook);
+    }
+  });
+
```

Comment:
Did you check this comment https://github.com/mozilla-sensorweb/sensorthings/pull/235#discussion_r97767450 ?